cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -121,23 +121,23 @@ function up() {
     fi
     eval ${_cli:?} run $params
 
-    # Copy k8s config and kubectl
+    ${_cli} scp --prefix $provider_prefix /etc/kubernetes/admin.conf - >${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
+
+    # Set server and disable tls check
+    export KUBECONFIG=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
+    kubectl config set-cluster kubernetes --server="https://$(_main_ip):$(_port k8s)"
+    kubectl config set-cluster kubernetes --insecure-skip-tls-verify=true
```

Comment:
We should use the copied kubectl so people don't need to have installed.

Reply:
We can't, because it is copied with root permissions and then a non root user can't use podman via the socket
See please comment on PR desc
It is fine to ask to have kubectl installed, it will be used for checking the cluster version